auth/auth_test.go

```diff
@@ -27,6 +27,11 @@ import (
 	"gopkg.in/square/go-jose.v2/jwt"
 )
 
+func NewTestAuthenticator(t testing.TB, dataStore sgbucket.DataStore, channelComputer ChannelComputer, opts AuthenticatorOptions) *Authenticator {
```

Comment:
Nitpick, but do we need t passed in here? Doesn't look to be used

Reply:
It's not used but added to prevent this function accidentally being moved and used in non-test code.

Reply:
Makes sense! Good shout! 😄